frontend/compositions/dashboardHeader/desktopNav.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react"
+import styles  from './header.module.css'
+
+export default function DesktopNav() {
+  let [selected, setSelected] = useState('Search');
+
+  const handleNavChange = (e: React.MouseEvent<Element>) => {
+    const value = (e.target as HTMLElement).innerText;
+    setSelected(value);
+  }
+
+  const setClassName = (linkName: string) => {
+    if (linkName === selected) {
+      return styles.tab;
+    }
+    return '';
```

Comment:
We could use a ternary to avoid duplicate return statements it you like! 
ala `return linkName === selected ? styles.tab : ''`